CoinDock-api/app/Http/Controllers/Auth/UserController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\V1\SignupRequest;
+use Illuminate\Http\Request;
+use App\Models\V1\User;
+use League\Uri\Http;
+
+class UserController extends Controller
+{
+    //
+
+    // protected function generateAccessToken($user){
+    //     $token= $user->createToken($user->email. '-'.now());
+    //     return $token->accessToken;
+    // }
+
+     /**
+     * Registration
+     */
+    public function register(SignupRequest $request)
+    {
```

Comment:
Can you change the function name to store() or sign()? It's not the register right.